src/crypto/ghash/ghash.c

```diff
@@ -43,6 +43,10 @@
 #if !defined( __LITTLE_ENDIAN) && defined(__DARWIN_LITTLE_ENDIAN)
 #define __LITTLE_ENDIAN __DARWIN_LITTLE_ENDIAN
 #endif
+#elif defined(LEDGER_VAULTHSM)
+//#include <machine/endian.h>
```

Comment:
This line could be removed (unless there is a good reason to keep it)

Reply:
Done